web/app/js/components/ServiceMesh.jsx

```diff
@@ -109,6 +118,15 @@ class ServiceMesh extends React.Component {
     ];
   }
 
+  getInstalledExtensions() {
+    const { extensions } = this.state;
+    let extensionList = [];
+    if (!_isEmpty(extensions.extensions)) {
+      extensionList = extensions.extensions.map(extension => extension.metadata);
```

Comment:
If we send back only the info we need from the backend, there shouldn't be a need to go through this extra step of filtering.

Reply:
Thanks for feedbacks, would check them out